### DISCUSSION TITLE
Where is $TORCH_COMMAND used?

### DISCUSSION THREAD

**Discussion by qpl23** [vladmandic/sdnext#193] [Q&A] [answered]
In webui.sh I see 

```export TORCH_COMMAND="pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/rocm5.2```

but I can't see this being used anywhere.  Looks like it should have triggered on my AMD setup, but in setup.log I see:

```
DEBUG [...] Running pip: install --upgrade torch torchaudio torchvision --index-url https://download.pytorch.org/whl/cu118
[...]  Torch repoorts CUDA not available
```

**Comment by vladmandic** [comment, marked as answer]
`TORCH_COMMAND` was ignored by installer, i've fixed it (will be on github in ~1h).
